iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/expected.inl

```diff
@@ -23,24 +23,15 @@ namespace iox
 {
 namespace cxx
 {
-namespace internal
-{
-template <typename... T>
-struct IsOptional : std::false_type
-{
-};
-template <typename T>
-struct IsOptional<iox::cxx::optional<T>> : std::true_type
-{
-};
-} // namespace internal
-
+// AXIVION Next Construct AutosarC++19_03-A12.1.5 : Delegating ctor doesn't fit-in here, as each initialization
+// for 'value' data member is in different style
```

Comment:
@elBoberido FYI, universal initialization `value{t}` not vanishes this warning. still reporting the `AutosarC++19_03-A12.1.5` violation.

Reply:
@saif-at-github I somehow assumed that this would not change the result but hoped for the best. At first I had to think a bit what you meant with the justification but now I think I have an idea. What do you think about rephrasing it to `This is a false positive since there is no fitting constructor available for delegation`?
If you think this would be more clear then it would also need to be changed for `error<T>`.